cmd/argocd/commands/project.go

```diff
@@ -120,33 +122,53 @@ func humanizeTimestamp(epoch int64) string {
 // NewProjectCreateCommand returns a new instance of an `argocd proj create` command
 func NewProjectCreateCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
 	var (
-		opts projectOpts
+		opts    projectOpts
+		fileURL string
+		upsert  bool
 	)
 	var command = &cobra.Command{
 		Use:   "create PROJECT",
 		Short: "Create a project",
 		Run: func(c *cobra.Command, args []string) {
-			if len(args) == 0 {
-				c.HelpFunc()(c, args)
-				os.Exit(1)
-			}
-			projName := args[0]
-			proj := v1alpha1.AppProject{
-				ObjectMeta: v1.ObjectMeta{Name: projName},
-				Spec: v1alpha1.AppProjectSpec{
-					Description:       opts.description,
-					Destinations:      opts.GetDestinations(),
-					SourceRepos:       opts.sources,
-					OrphanedResources: getOrphanedResourcesSettings(c, opts),
-				},
-			}
+			var proj v1alpha1.AppProject
 			conn, projIf := argocdclient.NewClientOrDie(clientOpts).NewProjectClientOrDie()
```

Comment:
do you need to get this on this line, can you wait until line 153?

Reply:
It is needed for unmarshaling in 139 and 141, though I noticed the connection below it could wait a bit.